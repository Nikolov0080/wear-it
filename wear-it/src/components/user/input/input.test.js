import React from 'react';
import renderer from 'react-test-renderer';
import Input from './input';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Input', () => {
    const tree = renderer
        .create(<BrowserRouter><Input 
            type="text" name='test' placeholder="meow"
            /></BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});