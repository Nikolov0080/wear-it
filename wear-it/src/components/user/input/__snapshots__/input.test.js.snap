// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Input 1`] = `
<div>
  <input
    autoComplete="off"
    className="input"
    name="test"
    onChange={[Function]}
    placeholder="meow"
    type="text"
  />
</div>
`;
